### ISSUE TITLE
[Photon/P1] 802.11n only mode not working on Photon/P1

### ISSUE THREAD

**Issue by technobly** [particle-iot/device-os#1556]
## Bug Report

802.11n only mode not working on Photon/P1

### Expected Behavior

Photon/P1 devices are built with a Cypress 43362 Wi-Fi radio that is designed to connect to 802.11b/g/n Wi-Fi networks.

### Observed Behavior

Tested on a ASUS RT-N66U and with 2.4GHz *802.11n only mode*, 0.6.0 connects, but 0.7.0-rc.1, 0.7.0 and 0.8.0-rc.8 do not connect.

### Steps to Reproduce

See above.

### Test App

You can use Tinker.

### References

https://community.particle.io/t/p0-wireless-n-only-mode-not-working-on-0-7-0-and-later-firmware/42748

https://community.particle.io/t/update-to-0-7-0-goes-wrong/40700/57

CH17929

**Comment by kasperkamperman** [comment]
Any milestone or plan when this bug will be fixed?

**Comment by DriftingShadows** [comment]
any updates on this? still doesn't work on 1.0.1

**Comment by Bacichetti** [comment]
Still no update on this issue??
How it's possible after so many months? How do we tell our clients that N mode does not work? It's on the specifications.

**Comment by kasperkamperman** [comment]
I'm also surprised that this is not a priority and a solution is included in the release candidate. My fear with the 3rd generation was that the 2nd generation (Photon), would be left alone. Which unfortunately seems the case.

**Comment by technobly** [comment]
@kasperkamperman @DriftingShadows @Bacichetti if you have the ability to test #1755 to see if it resolves the issue for y'all, your feedback is appreciated.
```
modules $ make clean all -s PLATFORM=photon COMPILE_LTO=n APP=tinker program-dfu
```

**Comment by kasperkamperman** [comment]
Of course I'd like to test. However what are the steps that I have to follow? Do I need to download a certain firmware and then run the make command?

**Comment by technobly** [comment]
@kasperkamperman I put more steps on a community post, but I just compiled some binaries for Photon and P1 for you to try. 
 This is compatible with 1.1.0-rc.1 compiled apps, or you can just use the included tinker binary.  You shouldn't need the bootloader, but just in case it's part of the natural build process so I zipped it all up.
https://www.dropbox.com/s/07rusi7uwe262wd/particle_device-os%401.1.0-rc.1-pr-1755.zip?dl=0

**Comment by Bacichetti** [comment]
I did test the fix here and it does fix the issue with 802.11n mode.
Thanks!

**Comment by kasperkamperman** [comment]
Thanks for the fix!
